src/tests.rs

```diff
@@ -46,6 +46,36 @@ fn test_bundle_keywords() {
     assert_that(&kw[2]).is_equal_to("c".to_string());
 }
 
+#[test]
+fn test_bundle_actions() {
+    let bun: Bundle = r#"{
+        "name": "aristotle",
+        "invocationImages": [],
+        "schemaVersion": "1.0-WD",
+        "version": "1.0.0",
+        "actions": {
+            "my_action": {
+                "modifies": true,
+                "stateless": true,
+                "description": "a custom action"
+            }
+        }
+    }"#
+    .parse()
+    .unwrap();
+
+    let actions = bun.actions;
+    assert_that(&actions).is_some();
```

Comment:
I think it might be easier to use the [assert_eq](https://doc.rust-lang.org/std/macro.assert_eq.html) macro here; we would just define a second "expected" bundle and pass it in.

I haven't used it, but [pretty_assertions](https://crates.io/crates/pretty_assertions) looks like a nice ux improvement on this with the same API.